client-v2/src/util/CAPIUtils.ts

```diff
@@ -98,9 +98,9 @@ const getPrimaryTag = (externalArticle: ExternalArticle): Tag | null =>
 
 const isLive = (article: CapiArticle) => {
   // `isLive` is `undefined` if item is from Live CAPI, so return `true`
-  // if we're from Preview CAPI, coerce `isLive` (string) field into a Boolean
+  // `CapiBool` is either a `string` or `boolean`, so cast it as a string and compare 🤢
   const isLiveField = article.fields.isLive;
-  return isLiveField === undefined || Boolean(isLiveField);
+  return isLiveField === undefined || isLiveField.toString() === 'true';
```

Comment:
What case does this catch that the other did and do we need to write a unit test for this given we can't get it right ??!?!

Reply:
The case where `isLiveField = 'false'` as `Boolean("false") === true` 🙄

Reply:
Incredible scenes

Reply:
I totally thought we tested that :(

Reply:
Yeah... we're explicitly testing it now though!